docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+version: '2.2'
```

Comment:
Could use some comments for new developers to know how this is different from the compose files in compose setup.